src/components/Containers/ContainerDetails.jsx

```diff
@@ -99,12 +100,18 @@ function ContainerDetails() {
             opType: actionType
         }).then((res) => {
             if (res) {
-                toast.success(res);
+                Swal.fire({
```

Comment:
I was not able to test this one case as I have an issue with .store.bin, possibly because of how I'm using colima -- I also had to symlink my docker.sock in order to start dockyard.